addons/sale_timesheet/data/sale_service_demo.xml

```diff
@@ -2,6 +2,10 @@
 <odoo>
     <data noupdate="1">
 
+        <record id="base.user_admin" model="res.users">
+           <field name="groups_id" eval="[(4, ref('analytic.group_analytic_accounting'))]"/>
+        </record>
```

Comment:
It does not seem to be part of this bug fix, right?

Reply:
I added this demo data in order to properly work some profitability related tour and test tour
Also it'll ease the testing for testing I guess 😅

Reply:
You changed the functional behavior in the demo data. Before doing that you should ask to Alexandra or change the tests according to your changes